account_invoice_view_payment/wizards/account_register_payments.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#        (https://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, api, models
+
+
+class AccountRegisterPayments(models.TransientModel):
+    _inherit = "account.register.payments"
+
+    @api.multi
+    def create_payment_and_open(self):
+        payment_model = self.env["account.payment"]
+        payments = payment_model
+        for payment_vals in self.get_payments_vals():
+            payments += payment_model.create(payment_vals)
+        payments.post()
+        res = {
+            "domain": [("id", "in", payments.ids), ("state", "=", "posted")],
+            "views": [(False, "form")],
+            "name": _("Payments"),
+            "view_type": "form",
```

Comment:
@kongrattapong FYI `view_type` is deprecated.

Reply:
@nikul-serpentcs Thank for review, I'll fix it.